jboss-kie-kieserver/added/launch/jboss-kie-kieserver-jms.sh

```diff
@@ -18,12 +21,14 @@ function prepareEnv() {
 function preConfigure() {
     KIE_JMS_FILE="${JBOSS_HOME}/standalone/deployments/ROOT.war/META-INF/kie-server-jms.xml"
     KIE_EJB_JAR_FILE="${JBOSS_HOME}/standalone/deployments/ROOT.war/WEB-INF/ejb-jar.xml"
+    KIE_AUDIT_PROPERTIES_FILE="${JBOSS_HOME}/standalone/deployments/ROOT.war/WEB-INF/classes/wildfly-jbpm.audit.jms.properties"
```

Comment:
Should KIE_JMS_FILE, KIE_EJB_JAR_FILE, KIE_AUDIT_PROPERTIES_FILE be unset in postConfigure(), since they were introduced in preConfigure()?

Reply:
Good point, will do this, thanks